modules/eco-poison-pill-about-watchdog.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-watchdog-devices_{context}"]
+= About watchdog devices
+
+Watchdog devices are of the following categories:
+
+* Independently powered hardware devices 
+* Hardware devices that share power with the hosts they control
+* Virtual devices implemented in software (or softdog)
+
+Hardware watchdog and softdog devices have electronic or software timers, respectively. These watchdog devices are used to ensure that the machine enters a safe state when an error condition is detected. The cluster is required to repeatedly reset the watchdog timer to prove that it is in a healthy state. This timer might elapse due to fault conditions, such as deadlocks, CPU starvation, and loss of network or disk access. If the timer expires, the watchdog or softdog device assumes that a fault has occurred and the device triggers a forced reset of the node.
+
+Hardware watchdog devices are more reliable than softdog devices.
+
+[id="understanding-pp-watchdog_{context}"]
+== Understanding Poison Pill Operator behavior with watchdog devices
+
+The Poison Pill Operator determines the remediation strategy based on the watchdog devices that are present. Since hardware watchdog devices are more reliable than softdog devices, the Operator prefers the former.
```

Comment:
anthropomorphization of Operator - "prefers". And I think this sentence is not required also. 
```suggestion
The Poison Pill Operator determines the remediation strategy based on the watchdog devices that are present.
```